lib/ex_machina/sequence.ex

```diff
@@ -22,15 +20,11 @@ defmodule ExMachina.Sequence do
        ExMachina.Sequence.reset
      end
   """
-  def reset do
-    Agent.update(__MODULE__, fn(_) -> Map.new end)
-  end
+  def reset, do: Agent.update(__MODULE__, fn(_) -> Map.new end)
```

Comment:
For this and `start_link` I think I prefer the previous version. This is a bit too busy IMO